tests/cases/rect.py

```diff
@@ -1,12 +1,16 @@
+"""This file implements a rectangle class """
+
 from dataclasses import dataclass
 
 
 @dataclass
 class Rectangle:
+    "Rectangle as a dataclass"
     height: int
     length: int
 
     def is_square(self) -> bool:
+        "Go likes this to be camel case"
```

Comment:
Could mention https://github.com/adsharma/py2many/issues/174 here.

Are any of the *_enum cases able to be accepted by golint by adding docstrings, or can one be adjusted to pass golint?  The enum values could have the case of their names changed - shouldnt effect anything  else.